cli/src/cli.rs

```diff
@@ -471,6 +470,10 @@ impl CliConfig {
         "http://127.0.0.1:8899".to_string()
     }
 
+    pub fn default_websocket_url() -> String {
+        "ws://127.0.0.1:8900".to_string()
```

Comment:
I was thinking we could do this:
https://github.com/solana-labs/solana-web3.js/blob/662bd3b405491cb8e7c4985b3fd506461231a890/src/connection.js#L709-L715
so if you set `--url`, then `--websocket_url` comes along for ride, unless explicitly overridden